plugins/jdg73/pom.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.radargun</groupId>
+      <artifactId>radargun-plugin-parent</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>plugin-jdg73</artifactId>
+   <name>JDG 7.3.x plugin for RadarGun</name>
+
+   <properties>
+      <version.infinispan>${jdg73.infinispan.version}</version.infinispan>
+   </properties>
+
+   <profiles>
+      <profile>
+         <id>jdg73-infinispan-version-default</id>
+         <activation>
+            <property>
+               <name>!jdg73.infinispan.version</name>
+            </property>
+         </activation>
+         <properties>
+            <version.infinispan>9.4.0.DR1-redhat-00001</version.infinispan>
+         </properties>
+      </profile>
+   </profiles>
+
+   <dependencies>
+      <dependency>
+         <groupId>org.radargun</groupId>
+         <artifactId>radargun-cache</artifactId>
+         <version>${project.version}</version>
+         <scope>provided</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.radargun</groupId>
+         <artifactId>radargun-query</artifactId>
+         <version>${project.version}</version>
+         <scope>provided</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.radargun</groupId>
+         <artifactId>radargun-mapreduce</artifactId>
+         <version>${project.version}</version>
+         <scope>provided</scope>
+      </dependency>
+
+      <dependency>
+         <groupId>org.radargun</groupId>
+         <artifactId>plugin-${plugins.infinispan-latest}</artifactId>
```

Comment:
@diegolovison I just saw this, but `jdg73` should depend on `jdg72`, and an infinispan plugin version. This is correct in the `jdg71` pom.xml https://github.com/radargun/radargun/blob/master/plugins/jdg71/pom.xml, but it is wrong in the `jdg72` pom file. Can you fix this?